staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3902,7 +3902,6 @@ type PodSpec struct {
 	// +optional
 	NodeName string `json:"nodeName,omitempty" protobuf:"bytes,10,opt,name=nodeName"`
 	// Host networking requested for this pod. Use the host's network namespace.
-	// If this option is set, the ports that will be used must be specified.
```

Comment:
I agree this current line was wrong / super confusing and should be dropped.

It *might* be useful to *add* a line describing how hostNetwork pods *should* specify ports they will use if they want scheduling to take that into account and avoid double-scheduling hostNetwork pods using the same host ports to the same node.

Reply:
I'll look at this.

Reply:
https://github.com/kubernetes/kubernetes/pull/131458 will have this, though I'm not quite sure yet how I want to phrase it, put up something to follow up on tomorrow.